src/room.py

```diff
@@ -1,2 +1,11 @@
 # Implement a class to hold room information. This should have name and
-# description attributes.
\ No newline at end of file
+# description attributes.
+
+class Room:
+    def __init__(self, name, description, equipment):
```

Comment:
here in init, you could consider doing something like '(self, name, description, equipment = [ ] )` , that way it is optional in the initialization and if the initialize command doesn't have the item list in its arguments, it will default it to an empty list.